packages/x-types/src/x-components-adapter.model.ts

```diff
@@ -47,4 +47,5 @@ export interface XComponentsAdapter {
   experienceControls: EndpointAdapter<ExperienceControlsRequest, ExperienceControlsResponse>
   aiQuestions: EndpointAdapter<AiQuestionsRequest, AiQuestionsResponse>
   aiTasks: EndpointAdapter<AiTasksRequest, AiTasksResponse>
+  aiOverviewSuggestions: EndpointAdapter<AiQuestionsRequest, Response>
```

Comment:
- Should it be `aiSuggestions` to decouple it from `overview`?
- Leave a `TODO` to replace `Response` in further PRs